index.js

```diff
@@ -3,16 +3,15 @@ const Mustache = require('mustache');
     let today = new Date().getTime() - 18000000;
     let firstDay = new Date(2021, 5, 19, 0, 0, 0, 0).getTime();
     let index = Math.floor((today - firstDay) / 864e5)
 
     return {
-        today: possibleSolutions[index],
-        yesterday: possibleSolutions[index - 1],
-        tomorrow: possibleSolutions[index + 1]
+        today: possibleSolutions[index + 1],
```

Comment:
This change seems unnecessary and out-of-scope. Please revert to using today as the `possibleSolutions[index]`, and subtracting 1 for yesterday and adding 1 for tomorrow.